lib/twitter-ads/error.rb

```diff
@@ -80,12 +69,13 @@ class NotFound < ClientError; end
   class BadRequest < ClientError; end
 
   class RateLimit < ClientError
-    attr_reader :reset_at, :retry_after
+    attr_reader :reset_at
 
     def initialize(object)
       super object
-      @retry_after = object.headers['retry-after']
-      @reset_at = object.headers['rate_limit_reset']
+      header = object.headers.fetch('x-account-rate-limit-reset', nil) ||
```

Comment:
The error object is only used once and thrown away? (Just want to be sure that we don't need some mechanism of refreshing values, e.g. continued errors from same line item object)

Reply:
@jbabichjapan The only purpose here is since `handle_rate_limit` option is `false` by default and users generally get this exception `RateLimit` from http.py, so we're giving the ability to manually inspect rate-limit-reset header and build some logic by themselves.

If users set `handle_rate_limit: true` then they shouldn't get this exception.